composer.json

```diff
@@ -59,6 +59,7 @@
     ],
     "minimum-stability": "dev",
     "require": {
+        "php": ">=7.1.0",
```

Comment:
```suggestion
        "php": "^7.1",
```

Reply:
7.2 since we'll need event dispatcher.

Reply:
So it official that Yii 3 will requires PHP 7.2?

Reply:
7.1 security supports ends on Dec 1st 2019, I guess it makes sense

Reply:
@rob006, yes.